cmd/util/environment_map.go

```diff
@@ -0,0 +1,14 @@
+package util
+
+import "strings"
+
+func LoadEnvironmentMap(environ []string) map[string]string {
```

Comment:
question: I find this function unnecessary. Why not just deal with the environment variables in the function where you want them. What is the point in putting them all in a dictionary and then passing that dictionary as an argument? If you want to make the installation of vscode plugins more reusable then I think it is better to have more arguments for it that are strictly related to installing plugins. Or put the required info in a struct and pass in a struct. But creating that struct or args should belong somewhere else. And then this function can be removed.